contracts/Manager.sol

```diff
@@ -633,7 +860,10 @@ contract Manager is UUPSOwnableUpgradeable, UsingRegistryUpgradeable {
         GroupWithVotes[] memory groupsWithVotes = new GroupWithVotes[](groups.length);
         uint256 totalVotes = 0;
         for (uint256 i = 0; i < groups.length; i++) {
-            uint256 votes = account.getCeloForGroup(groups[i]);
+            uint256 votes = specificStrategyVotedGroups.contains(groups[i])
+                ? account.getCeloForGroup(groups[i]) -
```

Comment:
Would it make things easier and cleaner to keep separate counts of CELO for the group according to default strategy and to the specific strategy?

Reply:
I believe that we would need to fill in historical data which I would probably try to avoid